### PR TITLE
Add entry in en-US locale and new ru-RU locale

### PR DESCRIPTION
**Pull request by scaledteam**

### CODE REVIEW

File: data/locale/en-US.ini [hunk 1]
```diff
@@ -2,4 +2,4 @@ GameCapture="Game Capture"
 CaptureCursor="Capture Cursor"
 CaptureWindow="Window"
 CaptureAnyWindow="Capture any window"
-AllowTransparency="Allow Transparency"
+AllowTransparency="Allow Transparency"
```
Comment (nowrep) [inline data/locale/en-US.ini:5]: This newline please

**Comment by scaledteam** [comment]
Sorry, my editor puts it automatically.

**Comment by scaledteam** [comment]
Probably i made a mess with some changes. If you want, you can just close pull request and copy `ru-RU.ini` manually.

**Comment by nowrep** [comment]
Thanks

**Comment by scaledteam** [comment]
Thank you for you great plugin!

**Pull request closed without merging**